Gemfile

```diff
@@ -3,7 +3,9 @@
 source 'https://rubygems.org'
 ruby '>= 2.3.0', '< 2.5.0'
 
-gem 'pkg-config'
```

Comment:
That all seems very reasonable. This seems to be a minimally invasive changeset as far as non-nanobox installs go, and there are no merge conflicts present, so I am going to approve this. 

If you haven't rebased and tested this lately I recommend doing so in case there is anything that you need to change that isn't obvious/a merge conflict. That way if you find anything you can add it to this PR before it's merged (: